src/Microsoft.Health.Dicom.Api/Features/Responses/MetadataResult.cs

```diff
@@ -19,7 +19,6 @@ public class MetadataResult : ObjectResult
         public MetadataResult(RetrieveMetadataResponse response)
             : base(response.ResponseMetadata)
         {
-            EnsureArg.IsNotNull(response, nameof(response));
```

Comment:
We might want to update the call to `base(response.ResponseMetadata)` to something like `base(response?.ResponseMetadata)`. I'm not sure of the implications without digging deeper, just something to think abou.t

Reply:
base(EnsureArg.IsNotNull(response, nameof(response).ResponseMetadata)

decided this would be best